.cruft.json

```diff
@@ -10,7 +10,7 @@
             "project_slug": "ravenpy",
             "project_short_description": "A Python wrapper to setup and run the hydrologic modelling framework Raven.",
             "pypi_username": "CSHS-CWRA",
-            "version": "0.6.0",
+            "version": "0.6.1",
```

Comment:
with all the changes that have been merged recently, I think that bumping the minor would probably make more sense though

Reply:
Bumping to 0.6.1 for debugging purposes. There are a handful of changes needed before we release a next minor version. Right now, the master branch relies on some git@master versions, and I'd like to sanitize that before we release a new version. This requires a game plan, so let's discuss over slack.